.github/workflows/rerun-workflows.yml

```diff
@@ -29,6 +29,12 @@ jobs:
           script: |
             const MAX_ATTEMPTS = 2;
             const ATTEMPT = ${{ github.event.workflow_run.run_attempt }};
+            const BRANCH = ${{ github.event.workflow_run.head_branch }};
+            const BRANCH_REF = context.ref;
+            const FAILED_RUN = ${{ github.event.workflow_run.html_url }};
+            const FAILED_WORKFLOW = ${{ github.event.workflow_run.name }};
+            const AUTHOR = ${{ github.event.workflow_run.head_commit.author.name }}
```

Comment:
I'm curious whether this will pick up the most recent commit, or the one this failed on - the final retry failure could be an hour after the commit, during which time the head commit could be different?